.eslintrc

```diff
@@ -370,7 +370,7 @@
       "always"
     ],
     "space-before-function-paren": [
-      "error",
+      "off",
```

Comment:
Let's remove this check if it's not needed! (Considering "Avoid having dead code in the  codebase")

Reply:
I think it needs to be explicitly turned off, but there should be a comment explaining it.